languages/cairo/generic/Parse_cairo_tree_sitter.ml

```diff
@@ -124,7 +124,7 @@ let rec map_qualified_name (env : env) (x : CST.qualified_name) :
 
```

Comment:
We need a test plan that actually shows something, some code that one can `-dump_ast` and see that now those annotations are getting resolved.